lib/galois-rpc/src/zero_knowledge_proof.rs

```diff
@@ -1,19 +1,15 @@
-use macros::model;
+use serde::{Deserialize, Serialize};
 
 // REVIEW: Are these fields fixed size?
-#[model(proto(raw(protos::union::galois::api::v3::ZeroKnowledgeProof), into, from))]
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct ZeroKnowledgeProof {
     #[serde(with = "::serde_utils::hex_string")]
-    #[debug(wrap = ::serde_utils::fmt::DebugAsHex)]
```

Comment:
seems useful? why remove this?

Reply:
No longer have access to our custom debug macro here. This will be replaced with a custom bytes type soon